apps/web/test/utils/bookingScenario/bookingScenario.ts

```diff
@@ -708,6 +746,7 @@ export function getOrganizer({
   selectedCalendars,
   destinationCalendar,
   defaultScheduleId,
+  weekStart = "Sunday",
```

Comment:
Explicitly setting the default here so we get the correct return, but could also set this in `TestData`.